tests/ha/remove_node.pm

```diff
@@ -40,7 +40,14 @@ sub remove_state_join {
     is_node(2) ? script_run("$crm_mon_cmd", $default_timeout) : save_state;
 
     # Second node needs to be reintegrated
-    assert_script_run("$join_cmd $node_01", 2 * $join_timeout) if is_node(2);
+    if (is_node(2)) {
+        type_string "$join_cmd $node_01 ; echo ha-cluster-join-finished-\$? > /dev/$serialdev\n";
+        if (check_screen('ha-cluster-join-password', 5)) {
```

Comment:
Same question WRT the timeout value. Also, now that both this join command and the one in `tests/ha/ha_cluster_join` would expect for the password prompt, it's a good candidate to refactor the code as a function in `lib/hacluster`.

Reply:
Yes, that's what I tried first but can take more time. Maybe after the BV?